python/bin/common.sh

```diff
@@ -21,7 +21,7 @@ if [ "$(uname)" = "Windows_NT" ]; then
 else
```

Comment:
Let's add a check for `DCOS_EXPERIMENTAL` and then use `: ${PYTHON:=python3${EXE}}`, otherwise let's keep `: ${PYTHON:=python3.5${EXE}}`.

Reply:
The idea with `DCOS_EXPERIMENTAL` is to control the minor version check, whether or not it should be a strict comparison or a "less than". This `python3.5` is only the default here, one can already overwrite it by exporting `$PYTHON`